sdk/eventgrid/Microsoft.Azure.EventGrid/src/Microsoft.Azure.EventGrid.csproj

```diff
@@ -1,21 +1,15 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
-	<!-- Please do not move/edit code below this line -->
-	<Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
-	<!-- Please do not move/edit code below this line -->
-
 	<PropertyGroup>
-		<PackageId>Microsoft.Azure.EventGrid</PackageId>
+	    <AssemblyTitle>Microsoft Azure EventGrid Library</AssemblyTitle>
 		<Description>This library can be used to publish events to Azure Event Grid and to consume events delivered by EventGrid. It also defines the event schemas for the events published to EventGrid by various Azure services. </Description>
-		<Version>3.1.0</Version>
-		<AssemblyName>Microsoft.Azure.EventGrid</AssemblyName>
+		<VersionPrefix>3.0.0</VersionPrefix>
 		<PackageTags>Microsoft Azure EventGrid;Event Grid;Event Grid Publishing;</PackageTags>
 		<PackageReleaseNotes>Added support for the new Azure Media Services event types for tracking state changes of Media Jobs and Media JobOutputs.  The existing MediaJobStateChangeEvent can be used to get all state change events for a Job but there are now separate events for each state like MediaJobScheduledEvent, MediaJobFinishedEvent, etc.  These allow subscribing for just the desired events.  Likewise a MediaJobOutputStateChangeEvent was added to get events for all MediaJobOutput state changes along with individual events for each JobOutput state.  MediaJobOutputProgressEvent was added to get events for Job output progress value, label and correlation data. This change also adds support for events related to Media LiveEvents to get notifications related to the ingestion of live streams (encoder connected, encoder disconnected, discontinuity, chunk dropped, etc).  Note JobState was renamed to MediaJobState to follow the Media prefix that all of the other Media Services Event classes use.  This is a breaking change, albeit a small one.</PackageReleaseNotes>
+	    <AzureApiTag>EventGrid_2019-02-01-preview;</AzureApiTag>
+        <PackageTags>$(PackageTags);$(CommonTags);$(AzureApiTag);</PackageTags>
 	</PropertyGroup>
 	<PropertyGroup>
-		<TargetFrameworks>$(SdkTargetFx)</TargetFrameworks>
+    <NoWarn>$(NoWarn);CS0618</NoWarn>
```

Comment:
Where did this NoWarn come from? Is is still necessary?

Reply:
that was due to a warning that results in an error on the build pipeline. I added it to get it passing. Seems NewtonSoft.Json needs to be updated to version 10.0.3
```Customization\JsonSerializerExtensions.cs(34,17): warning CS0618: 'JsonSerializerSettings.ReferenceResolver' is obsolete: 'ReferenceResolver property is obsolete. Use the ReferenceResolverProvider property to set the IReferenceResolver: settings.ReferenceResolverProvider = () => resolver' [C:\git\azure-sdk-for-net\src\sdks\eventgrid\DataPlane\Microsoft.Azure.EventGrid\Microsoft.Azure.EventGrid.csproj]```

Reply:
We should be very careful about just ignoring these obsolete warnings. The fact that it is a new warning actually leads me to believe we might have changed the version of Newtonsoft.Json, so I suggest adding the no warning but also filing an issue for this so someone can make the right decision about whether or not to switch versions or update the code to address this issue.